### DISCUSSION TITLE
Docker in Docker like with rancher destkop and nerdctl ?

### DISCUSSION THREAD

**Discussion by florentulve** [rancher-sandbox/rancher-desktop#4027] [General]
Hi all,

Considering using Rancher Destkop, I have a workflow involving Docker in Docker.
With Docker Deestkop I share the docker.sock.
Is there a way to achieve something similar with rancher destkop / nerdctl ?

Thanks !

**Comment by jandubois** [comment]
You should be able to share the `/var/run/docker.sock` into containers, if that is what you are asking for. Did you run into problems?
